Bowling/Bowling/FrameWithBonusRollFactory.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BowlingApp
+{
+    public class FrameWithBonusRollFactory
+    {
+        public const int totalPinsCount = 10;
```

Comment:
Ditto

Reply:
Removed public variable and used a constructor to set this.